project/task1/detectFixations.py

```diff
@@ -0,0 +1,29 @@
+import statistics
+from pointClass import Point
+from handleWindowPoints import handleWindowPoints
+
+def detectFixations(rowData, DT, SC):
+    fixationDuration = []
+    fixationCentroid = []
+    i=0
+    while i+SC-1 <= len(rowData)-1:
```

Comment:
why `- 1` on both side of the statement?

Reply:
noticed